Dockerfile

```diff
@@ -0,0 +1,29 @@
+# gammapy docker container
+FROM continuumio/miniconda3:latest
+RUN apt-get update && apt-get install -y build-essential python3-pip
+
+# define workdir
+WORKDIR /usr/src/app/
+
+# copy content
+COPY . /usr/src/app/
+
+# create conda env
+RUN conda install -c conda-forge mamba
+RUN mamba env create -n gammapy -f environment-dev.yml
+RUN echo 'source activate gammapy' >> ~/.bashrc
+ENV PATH /opt/conda/envs/gammapy/bin:$PATH
+
+# install gammapy
+RUN pip install -e .
+
+# download datasets
+RUN gammapy download datasets
```

Comment:
The datasets size is 192MB out of 3GB of the total image size. We ship the datasets within the docker image together with the Gammapy code and tutorials, so to provide the possibility to play with the tutorials in shipped Jupyterlab. See [here](https://github.com/gammapy/gammapy/pull/3339/files#diff-53d09420b94e4eb52ff6c09b64cfea1439caaa8f2dd67c796cc48e9958500e89R32).

Reply:
The image is 3Gb??? That is HUGE

Reply:
This is interesting, I have never checked what the actual size of the dev environment is. Do you have any idea why it is so large?

Reply:
Conda envs get huge fast since they duplicate many system libraries and include e.g. the mal by default. 

Adding nomkl reduces conda env quite a bit.

Reply:
just for info

```
$ du -hs /opt/miniconda3/envs/gammapy-0.18.2
1,2G	/opt/miniconda3/envs/gammapy-0.18.2
```